VALUES.md

```diff
@@ -0,0 +1,30 @@
+# Ayo.js Community Values
+
+We don't care how you pronounce it: be it "ey-yo", "I-oh", or "Awooooo" -- the Ayo.js project strives to create an environment where you can feel at home and focus on your contributions to this nice shared space that we all benefit from.
+
+Ayo.js is about the humans that help make it happen. It's a project aimed at creating a new foundation of project governance and management that brings **humans** and their concerns front and center.
+
+## Ayo.js is more than the code in its repository.
+
+Documentation, long-term support, translations and localizations, package management, libuv, v8 and Chakra, builds/ops, testing and its infrastructure, hardware and IoT, Android, streams, community events, facilitation, moderation and education. These and more are all concerns that can be handled separately and by members who might be much better qualified for their individual tasks than a central decision body.
+
+Ayo.js wants to go further than Node Core did in spreading responsibility across the project, so everyone can focus and specialize better than before. Contributors should be able to organize and manage their own subprojects in a way that is effective and efficient for their individual unique needs.
+
+## Humans are more important than red tape.
+
+If governance policy is not serving the values set forth here, that governance policy should change with the required agility. Following the governance policy to the letter should not take precedence over staying true to our stated values, not to mention the well-being of our humans.
+
+## Humans are more important than APIs and Benchmarks.
+
+Technical discussions are difficult, especially in large communities with members with all sorts of perspectives. You're bound to find yourself in situations with no clear consensus, with new concerns being brought up on a regular basis, and things that might seem insignificant in other contexts suddenly explode.
+
+The Ayo.js project's perspective is that when this happens, it often takes a toll on the well-being of the members participating, or it prevents potential members from contributing again, if at all. It also recognizes that the difference between individual choices is less important than the ability to come to a perfect technical decision.
```

Comment:
i really can't parse the last sentence here? i feel like this could be elaborated better